### PR TITLE
MM-51362 - Fix proximity notice engaging after call ended

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- By manually turning off the speakerphone, we were inadvertently starting the proximity sensor/notice (i.e., the screen would turn dark, to prevent the caller from pressing buttons with their face while on a call). 
- Also updating the in-call manager while we're at it.
- Found this while investigating the racy connection issue (https://mattermost.atlassian.net/browse/MM-49663)

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-51362

#### Checklist
- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
- Android: 13, Pixel 6
- iOS: 15.7, iPhone 7 plus

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by streamer45 — APPROVED** [review]
Nice find :+1:

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "react-native-hw-keyboard-event": "0.0.4",
     "react-native-image-picker": "5.0.2",
     "react-native-in-app-review": "4.2.1",
-    "react-native-incall-manager": "github:cpoile/react-native-incall-manager",
+    "react-native-incall-manager": "github:cpoile/react-native-incall-manager#634c8fc",
```
Comment (streamer45) [inline package.json:72]: Could you remind me why we are using a fork?

Comment (cpoile) [reply package.json]: @streamer45 I'm trying to think why, it was back during the first v2 move, but I can't remember, sorry. Removing.

Comment (streamer45) [reply package.json]: Cool, one less thing to keep updated :)

**Review by streamer45 — APPROVED** [review]

**Review by enahum — APPROVED** [review]

**Pull request merged**